src/app/organisation/index.njk

```diff
@@ -65,16 +80,48 @@
             $("#organisation_table_row tr").filter(function () {
               $(this).toggle($(this).text().toLowerCase().indexOf(value) > -1)
             });
+            
           });
+        
+          $("#searchUsers").on("click", function () {  
+                searchUser=document.createElement('input');
+                searchUser.type='hidden';
+                searchUser.name='searchUser';
+                searchUser.value = $("#organisation-search").val().toLowerCase();
+                document.getElementById('organisation').appendChild(searchUser);
+                document.getElementById('organisation').submit();  
+            });     
+         
         });
+
       </script>
 
-      <div class="govuk-grid-column-full">
-        <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--xl govuk-!-margin-top-0">
-        <p class="govuk-body-m">Search for people within the organisation,  {{ cookie.getOrganisationName() }}</p>
-        <input class="govuk-input govuk-input--width-20" id="organisation-search" name="organisation-search" type="text" placeholder="Search" style="margin-bottom: 10px">
-      </div>
+      {% if currentPage > 1 %}
+      <script>
+            /** If user click on next page then page will scroll to users table area */
+            $(document).ready(function () {
+            $('html, body').animate({
+              scrollTop: ($('#organisation_table').offset().top )
+              }, 0); 
+            });
+      </script>
+      {% endif %}
+
       <form id="organisation" action="/organisation" class="form-group" method="post" role="form" >
+        <div class="govuk-grid-column-two-thirds">    
+          <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--xl govuk-!-margin-top-0">
+          <p class="govuk-body-m">Search for people within the organisation,  {{ cookie.getOrganisationName() }}</p>
+          <input class="govuk-input govuk-input--width-20" id="organisation-search" name="organisation-search" type="text" value="{{ searchUserName }}" placeholder="Search">
+          
```

Comment:
Minor: You can submit the form without any value, which does not make a lot of sense.
* leads to endless load times when the app has items in the form.
* Otherwise, if there are no items, it loads very quickly when search blank

Reply:
![Uploading Screenshot 2024-08-06 at 15.47.05.png…]()

Reply:
![Uploading Screenshot 2024-08-06 at 15.47.05.png…]()

Reply:
<img width="1440" alt="Screenshot 2024-08-06 at 15 47 05" src="https://github.com/user-attachments/assets/7a91352b-1797-4ee6-8c63-71211d02aca9">

Reply:
It was loading endlessly  because of '<button>' tag. so changed it to <a>.

Reply:
Oh, cool,  out of curiousity, why is that the case?